samples/sample-resource-deployment/package.json

```diff
@@ -23,6 +23,98 @@
   ],
   "contributes": {
     "resourceDeploymentTypes": [
+      {
+				"name": "validations-wizard",
+				"displayName": "%validation.wizard.display.name%",
+				"description": "%validation.wizard.description%",
+				"platforms": "*",
+        "icon": {
+          "light": "./images/book.svg",
+          "dark": "./images/book_inverse.svg"
+        },
+				"providers": [
+					{
+						"notebookWizard": {
+              "notebook": "%deployment-notebook-1%",
+							"runNotebook": false,
+							"codeCellInsertionPosition": 1,
+							"actionText": "%deploy.wizard.action%",
+							"title": "%wizard.new.wizard.title%",
+							"name": "wizard.new.wizard",
+							"labelPosition": "left",
+							"generateSummaryPage": false,
+							"pages": [
+								{
+									"title": "%wizard.select.cluster.title%",
+									"sections": [
+										{
+											"fields": [
+                        {
+                          "label": "%arc.cores-limit.label%",
+                          "description": "%arc.sql.cores-limit.description%",
+                          "variableName": "AZDATA_NB_VAR_SQL_CORES_LIMIT",
+                          "type": "number",
+                          "defaultValue": 5,
+                          "required": false,
+                          "validations": [
+                            {
+                              "type": "is_integer",
+                              "description": "%cores.limit.should.be.integer%"
+                            },
+                            {
+                              "type": ">=",
+                              "target": "AZDATA_NB_VAR_SQL_CORES_REQUEST",
+                              "description": "%cores.limit.greater.than.or.equal.to.requested.cores%"
+                            }
+                          ]
+                        },
+                        {
+                          "label": "%arc.cores-request.label%",
+                          "description": "%arc.sql.cores-request.description%",
+                          "variableName": "AZDATA_NB_VAR_SQL_CORES_REQUEST",
+                          "type": "number",
+                          "defaultValue": 2,
+                          "required": false,
+                          "validations": [
+                            {
+                              "type": "is_integer",
+                              "description": "%requested.cores.should.be.integer%"
+                            },
+                            {
+                              "type": "<=",
+                              "target": "AZDATA_NB_VAR_SQL_CORES_LIMIT",
+                              "description": "%requested.cores.less.than.or.equal.to.cores.limit%"
+                            }
+                          ]
+                        }
+											]
```

Comment:
The indentation seems off here - make sure you're using consistent spacing for this file

Reply:
good catch. Converted all indentation from spaces to tabs to make it consistent with other files.

Reply:
done